docs/recovery.md

```diff
@@ -0,0 +1,351 @@
+}
+
+⛰  lncli --network=simnet exportchanbackup --all --output_file=channels.backup
+```
+
+As shown above, a user can either: specify a specific channel to backup, backup
+all existing channels, or backup directly to an on-disk file. All backups use
+the same format.
+
+#### Streaming Updates via `SubscribeChannelBackups`.
+
+Using the gRPC interace directly, [a new call:
+`SubscribeChannelBackups`](https://api.lightning.community/#subscribechannelbackups).
+This call allows users to receive a new notification each time the underlying
+SCB state changes. This can be used compared to the file-system approach to
+implement more complex backup schemes.
+
+### Recovering Using SCBs
+
+If a node is being created from scratch, then it's possible to pass in an
+existing SCB using the `lncli create` or `lncli unlock` commands:
+```
+⛰  lncli create -multi_file=channels.backup
+```
+
+Alternatively, the `restorechanbackup` command can be used if `lnd` has already
+been created at the time of SCB restoation:
+```
+⛰   lncli restorechanbackup -h
+NAME:
+   lncli restorechanbackup - Restore an existing single or multi-channel static channel backup
+
+USAGE:
+   lncli restorechanbackup [command options] [--single_backup] [--multi_backup] [--multi_file=]
+
+CATEGORY:
+   Channels
+
+DESCRIPTION:
+
+  Allows a user to restore a Static Channel Backup (SCB) that was
+  obtained either via the exportchanbackup command, or from lnd's
+  automatically manged channels.backup file. This command should be used
+  if a user is attempting to restore a channel due to data loss on a
+  running node restored with the same seed as the node that created the
+  channel. If successful, this command will allows the user to recover
+  the settled funds stored in the recovered channels.
+
+  The command will accept backups in one of three forms:
+
+     * A single channel packed SCB, which can be obtained from
+       exportchanbackup. This should be passed in hex encoded format.
+
+     * A packed multi-channel SCB, which couples several individual
+       static channel backups in single blob.
+
+     * A file path which points to a packed multi-channel backup within a
+       file, using the same format that lnd does in its channels.backup
+       file.
+
+
+OPTIONS:
+   --single_backup value  a hex encoded single channel backup obtained from exportchanbackup
+   --multi_backup value   a hex encoded multi-channel backup obtained from exportchanbackup
+   --multi_file value     the path to a multi-channel back up file
+```
+
+Once the process has been initiated, `lnd` will proceed to:
+
+  1. Given the set of channels to recover, the server will then (using the new
+     chanbackup) package, will insert a series of "channel shells" into the
```

Comment:
Don't think we really need to mention the `chanbackup` package. Can be reworded to: 

`Given the set of channels to recover, the daemon will insert a series of "channel shells" into the database.`